### ISSUE TITLE
Allow different names for `test` to accommodate `expect`"

### ISSUE THREAD

**Issue by daenamkim** [playwright-community/eslint-plugin-playwright#270]
Hi

I have been struggling to figure out how to solve an issue when I use two `test`s; one for `test`, the other is `testWeb3`. Both basically using the same `test` of playwright but just different names and they were extended.

_test_
<img width="1229" alt="Screenshot 2024-03-01 at 15 59 38" src="https://github.com/playwright-community/eslint-plugin-playwright/assets/16159767/98bd09bf-8236-4554-a2ac-4b2f2455047e">

_testWeb3_
<img width="1291" alt="image" src="https://github.com/playwright-community/eslint-plugin-playwright/assets/16159767/f7f751b0-bac2-4aa3-98a9-1332ed88dd95">

---

The issue is in the case that I use both `test`s in the same test file.

### Error cases 1

```typescript
import { Page, expect, test, testWeb3 } from './fixtures/auth';

test('1 should be 1', async () => {
  await expect(1).toBe(1);
});

testWeb3('2 should be 2', async () => {
  await expect(2).toBe(2); // ERROR
});
```

```sh
  13:9  error  Expect must be inside of a test block  playwright/no-standalone-expect

✖ 1 problem (1 error, 0 warnings)
```

### Error cases 2

```typescript
testWeb3.beforeEach(async () => {
  await 1;
});

test('1 should be 1', async () => {
  await expect(1).toBe(1);
});

testWeb3('2 should be 2', async () => {
  await expect(2).toBe(2); // ERROR
});
```

```sh
  17:9  error  Expect must be inside of a test block  playwright/no-standalone-expect

✖ 1 problem (1 error, 0 warnings)
```

### Success? case 1

```typescript
test.beforeEach(async () => {
  await 1;
});

test('1 should be 1', async () => {
  await expect(1).toBe(1);
});

testWeb3('2 should be 2', async () => {
  await expect(2).toBe(2);
});
```

```sh
✨  Done in 8.84s.
```

### Success? case 2

```typescript
test.beforeAll(async () => {
  await 1;
});

test('1 should be 1', async () => {
  await expect(1).toBe(1);
});

testWeb3('2 should be 2', async () => {
  await expect(2).toBe(2);
});
```

```sh
✨  Done in 8.82s.
```

I really have no idea why `test.beforeAll` and `test.beforeEach` are ok and how to solve this issue without using `beforeAll` and `beforeEach`. 😰

**Comment by mskelton** [comment]
@daenamkim The `globalAliases` setting is exactly what you are looking for:

https://github.com/playwright-community/eslint-plugin-playwright#aliased-playwright-globals
